src/components/tabs/js/tabsController.js

```diff
@@ -169,6 +170,8 @@ function MdTabsController ($scope, $element, $window, $mdConstant, $mdTabInkRipp
 
   function handleMaxTabWidth (newWidth, oldWidth) {
     if (newWidth !== oldWidth) {
+      var elements = getElements();
```

Comment:
Alright, makes sense. It starts querying from the element anyway, so it shouldn't be too much of a difference.